homework/12_V/dom circle intersection/main.py

```diff
@@ -0,0 +1,73 @@
+# Nikola Milkov 12v №22
```

Comment:
You shouldn't have commited the `.idea` folder. Your preferred IDE and its settings are not interesting to me.

Reply:
Also you've commited in the wrong folder. You should have commited in `homework/12_V/22_Nikola_Milkov/homework01/`.

Anyway. I will move your files to the correct directory.